main.go

```diff
@@ -142,23 +168,56 @@ func getPkgs(pkg string) ([]string, error) {
 }
 
 // coverage runs test for the given pkg and returns cover profile.
-func coverage(pkg string, optionalArgs []string, v bool) ([]*cover.Profile, error) {
-	f, err := ioutil.TempFile("", "goverage")
+// success indicates "go test" succeeded or not. coverage may return profiles
+// even when success=false. When "go test" fails, coverage outputs "go test"
+// result to stdout even when verbose=false.
+func coverage(pkg string, optArgs []string, verbose bool) (profiles []*cover.Profile, err error, success bool) {
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error should be the last type when returning multiple items